src/dialog-renderer.js

```diff
@@ -124,6 +122,11 @@ export class DialogRenderer {
       });
     };
 
+    dialogController.centerDialog = () => {
```

Comment:
Let's just `dialogController.centerDialog = centerDialog` here so it uses hte same function.